commit 669eba7cd36f83b94672abc04438ddaea2a85902
Author: Youri Westerman <tetracon@gmail.com>
Date:   Mon Apr 26 23:40:05 2021 +0200

    Added oars content rating to appdata xml (https://hughsie.github.io/oars/)
    Added screenshots to appdata xml
    Added release information to appdata xml

diff --git a/share/metainfo/org.qt-project.qtcreator.appdata.xml b/share/metainfo/org.qt-project.qtcreator.appdata.xml
index 4706e69b22..888bbb9a8e 100644
--- a/share/metainfo/org.qt-project.qtcreator.appdata.xml
+++ b/share/metainfo/org.qt-project.qtcreator.appdata.xml
@@ -26,4 +26,19 @@
 
  <url type="homepage">https://www.qt.io/ide/</url>
  <project_group>Qt</project_group>
+ <content_rating type="oars-1.1" />
+ <screenshots>
+  <screenshot>
+   <image>https://doc.qt.io/qtcreator/images/qtcreator-breakdown.png</image>
+  </screenshot>
+  <screenshot>
+   <image>https://doc.qt.io/qtcreator/images/qtcreator-qt-quick-editors.png</image>
+ </screenshots>
+ <releases>
+  <release version="4.14.2" date="2021-03-22">
+   <description>
+    <p>Qt Creator v4.14.2</p>
+   </description>
+  </release>
+ </releases>
 </component>
